content/credits.yaml

```diff
@@ -24,8 +24,7 @@ team:
   - Ashwin Ramaswami
   - Benjamin Swerdlow
   - Jason Telanoff
-# Add yourself here when you first submit a contribution.
-
+  - Ayush Bherwani
```

Comment:
I believe that you should not remove information that people should add themselves here whe submitting a first commit.

Reply:
I'll do the needful changes.